### PR TITLE
feat: viewer watching page

### PR DESCRIPTION
**Pull request by faiq-naufal**
# Description
This PR relates this issue #16. This PR will enable a page that can be used by other users to watch the live streaming session from a specific streamer. 

- The page can be accessed through the URL `/streaming/watch/{STREAM_ID}`
- The video player we currently use to play the DASH or HLS formats is [shaka-player](https://github.com/shaka-project/shaka-player).
- The viewer page consists of three main panels:
   - Video panel: panel that displays the video player plays the live streaming video broadcasted by the streamer
   - Information panel: the panel that displays the information data about the specific stream such as the stream title and description.
   - Activity panel: the panel that displays the activity log from the user. This is just UI only for now and is still not usable.

Additional things:
- Create lozenge UI component
- Create a viewer counter UI component. This is just the UI and the component is still not usable for now.

### CODE REVIEW

File: src/features/viewer/room/app-viewer-room.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        top: 0;
+        right: 0;
+        width: 20rem;
+        height: 100%;
+        max-height: 100vh;
+        border-left: 1px solid #d1d5db;
+      }
+    }
+
+    @media (min-width: 1280px) {
```
Comment (ingrid011) [inline src/features/viewer/room/app-viewer-room.js:79]: just a question mas @faiq-naufal , for 1280px means for desktop, but not necessary all styling need to add this break point right? This is for adjusting the responsiveness? I mean, could you advice more why need this break point?

Comment (faiq-naufal) [reply src/features/viewer/room/app-viewer-room.js:79]: Yes, that's true this `1280px` breakpoint is not necessary for all styling. This is just adjusting the design of the side bar/panel for the viewer page. See these example below:

If we follow the design on the figma, on `1024px` the design should be like this. The side bar/panel will have width of `23rem`
Screen Shot 2022-10-18 at 09 14 50

You may notice that the side bar/panel is quite wide on that screen. Therefore, I revised the design to make it less wider and change the width to `20rem`. This is the result on `1024px`
Screen Shot 2022-10-18 at 09 14 59

Do you think the result is better? Because it is for me. I need your opinion. 

So, basically on `1024px` the design will be like the 2nd picture above with width of `20rem`. And on the `1280px` the design will be adjusted to follow the actual design on the Figma with width of `23rem`

Comment (ingrid011) [reply src/features/viewer/room/app-viewer-room.js:79]: @faiq-naufal I see.. I think it's better for the 2nd picture since if I'm as a viewer, I prefer to see the video screen larger that the chat column (video is more important because we go to this page to watch right? That's the motivation as viewer).
So it's okay like that, mas. Thank you for the explanation.

**Review by ingrid011 — APPROVED** [review]
LGTM
